code/workspaces/web-app/src/PrivateApp.js

```diff
@@ -19,7 +19,8 @@ const { projectPermissions: { PROJECT_STORAGE_LIST, PROJECT_STACKS_LIST, PROJECT
 const PrivateApp = ({ promisedUserPermissions }) => (
   <NavigationContainer userPermissions={promisedUserPermissions.value}>
     <Switch>
-      <Route exact path="/" component={LandingPage} />
+      <Route exact path="/projects" component={ProjectsPage} />
+      <Redirect exact from="/" to="/projects" />
```

Comment:
I'm not sure I like this redirect. I was expecting the `AuthCallback` component to be updated here https://github.com/NERC-CEH/datalab/blob/master/code/workspaces/web-app/src/components/app/AuthCallback.js#L22